<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusecMainWindow</class>
 <widget class="QMainWindow" name="MusecMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>260</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Musec</string>
  </property>
  <property name="windowIcon">
   <iconset resource="musec.qrc">
    <normaloff>:/icon/musec.ico</normaloff>:/icon/musec.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnPlay">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>100</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkTitle">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>396</x>
      <y>34</y>
      <width>14</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblAlbum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>400</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Album:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInfo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>300</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkAlbum">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>396</x>
      <y>134</y>
      <width>14</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>400</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Title:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edAlbum">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>380</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>100</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>Accept</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblArtist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>400</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Artist:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkArtist">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>396</x>
      <y>84</y>
      <width>14</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edTitle">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>380</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblScore">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>90</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Score: 0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edArtist">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>380</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMusic">
    <property name="title">
     <string>Music</string>
    </property>
    <addaction name="actAddDir"/>
    <addaction name="actAddFiles"/>
    <addaction name="actClear"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actStats"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actAbout"/>
   </widget>
   <addaction name="menuMusic"/>
   <addaction name="menuInfo"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actAddDir">
   <property name="text">
    <string>Add Directory...</string>
   </property>
  </action>
  <action name="actAddFiles">
   <property name="text">
    <string>Add Files...</string>
   </property>
  </action>
  <action name="actClear">
   <property name="text">
    <string>Clear List</string>
   </property>
  </action>
  <action name="actStats">
   <property name="text">
    <string>Stats</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="musec.qrc"/>
 </resources>
 <connections/>
</ui>
